docs/usersGuide/reproduceResults.rst

```diff
@@ -0,0 +1,65 @@
+
+Here an example:
+
+.. code-block:: bash
+
+    /path/to/roddy.sh rerun \
+        config.WGS@alignment \
+        pid1,pid2 \
+        --useconfig=/path/to/configs/applicationProperties-analysis-local-lsf.ini \
+        --usefeaturetoggleconfig=/path/to/configs/featureToggles.ini \
+        --usePluginVersion=AlignmentAndQCWorkflows:1.2.73-0 \
+        --configurationDirectories=/path/to/configs \
+        --useiodir=/path/to/pidDir/,/path/to/outputDir \
+        --cvalues=fastq_files:/path/to/fastqs/r1.fq.gz;/path/to/fastqs/r2.fq.gz \
+        --useRoddyVersion=3.3.3
+
+The content of the file is a one-liner, but here it is broken down into arguments for readability. You'll notice the ``--cvalues`` parameter and that it contains a ';' used in the fastq_files configuration value to delimit read 1 and 2 FASTQs. The content of the configuration values can be arbitrary and is completely up to the workflow developer. In this case, the semicolons is als statement terminator in Bash and will be interpreted as such, unless quoted or escaped. Thus, the fix here is of course to quote the value of the parameter:
+
+.. code-block:: bash
+
+    --cvalues='fastq_files:/path/to/fastqs/r1.fq.gz;/path/to/fastqs/r2.fq.gz' \
+
+Same Analysis on Different Input Data
+-------------------------------------
+
+If you want to extend your analyses with new input data you can start with an old Roddy call and adapt it. Often used adaptations are
+
+  * change the output directory
+  * change the input directory
+  * change the applicationProperties.ini: if you intend to change the cluster configuration or configuration file directories
+  * change configuration values that are metadata of the workflow specifically derived from the analysed sample (e.g. insert size distribution parameters for the sample)
+
+Note that these changes can be quite tedious and error prone to be done manually. Usually it is best to write a small script -- in particular if you have to run your analyses on many new input datasets.
+
+
+Beware: Multiple Execution Stores
+---------------------------------
+
+Beware that you may have multiple execution store directories if Roddy was run with in "rerun" mode, e.g. to complete a failed job. Here things can get really complicated if the different parts of the output data were produced with different versions or configurations.
+
+  * Ideally you should call Roddy with explicitly mentioning the plugin version, like in the example above. Due to the plugin dependency and automatic loading you may want to check that all used plugins were identical. You can check the ``versionInfo.txt`` file in the execution stores to learn which versions were used during each execution.
```

Comment:
Ideally, you should call Roddy explicitely mentioning?

Reply:
What do you mean? Is the sentence incomprehensible?